File: src/main/kotlin/ympa_kotlin_wiremock_server/models/WarehouseStockDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class WarehouseStockDTO(
    @field:JsonProperty("type")
    val type: WarehouseStockType,

    @field:JsonProperty("count")
    val count: kotlin.Long,

)

File: lib/app/Models/WarehouseStockDTO.php
<?php
/**
 * WarehouseStockDTO
 */
namespace app\Models;

/**
 * WarehouseStockDTO
 * @description Информация об остатках товара.
 */
class WarehouseStockDTO {

    /** @var string $type */
    public $type = "";

    /** @var int $count Значение остатков.*/
    public $count = 0;

}
